src/blant.c

```diff
@@ -41,8 +42,8 @@ int _quiet; // suppress notes and/or warnings, higher value = more quiet
 
 char * _sampleFileName;
 
-// _k is the global variable storing k; _Bk=actual number of entries in the canon_map for given k.
-unsigned int _k, _min_edge_count;
+// _k is the global variable storing k; _Bk=actual number of entries in the canon_map for given k; _k_base is the k that we will be using for the base of smaller canon map.
```

Comment:
Right. So assuming this comment is correct, then we only need to load the k=8 canon_maps (in the mapCanonMap stuff above) if k>8. But please run the full regression tests and let me know what happens.

Reply:
I should've been more clear in the comment about the definition of _k_base

Reply:
Updated the comment